src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -25,35 +25,74 @@ class PyTestResult(TestResult):
 
 # TODO: Support deps
```

Comment:
Can drop these comments if you end up doing another CI burn (if not, don't worry about it).